src/js/common/schemaform/FormApp.jsx

```diff
@@ -68,8 +71,7 @@ export default class FormApp extends React.Component {
           {formConfig.title && !isIntroductionPage && <FormTitle title={formConfig.title} subTitle={formConfig.subTitle}/>}
           {content}
         </div>
-        <div className="usa-width-one-third medium-4 columns show-for-medium-up">
-        </div>
+        <div className="columns usa-width-one-third medium-4"><AskVAQuestions/></div>
```

Comment:
By including this here, we are including in all forms that use this component, including pension forms, is that within the scope of these changes?